tests/test_pastasoss.py

```diff
@@ -1,21 +1,12 @@
+import numpy as np
```

Comment:
I see we have a `pastasoss/tests/test_pastasoss.py` and this file `tests/test_pastasoss.py`. Are they both supposed to be there?

Reply:
This has been fixed and now all test should be associated with `tests/test_pastasoss.py`.